apps/angular/pipe-easy/src/app/pipes/join.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { JoinPipe } from './join.pipe';
+
+describe('JoinPipe', () => {
+  it('create an instance', () => {
+    const pipe = new JoinPipe();
+    expect(pipe).toBeTruthy();
+  });
+});
```

Comment:
If you keep this file, you should write some useful test. Otherwise the file is unnecessary